app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
```

Comment:
The `people` dictionary is defined as a class variable. This is fine for storing all instances, but be cautious about potential memory issues if the list of people grows large.